action/protocol/staking/read_state.go

```diff
@@ -101,6 +101,21 @@ func readStateBucketByIndices(ctx context.Context, sr protocol.StateReader,
 	return pbBuckets, height, err
 }
 
+func readStateBucketCount(ctx context.Context, csr CandidateStateReader,
+	_ *iotexapi.ReadStakingDataRequest_BucketsCount) (*iotextypes.BucketsCount, uint64, error) {
+	total, err := getTotalBucketCount(csr.SR())
+	if errors.Cause(err) == state.ErrStateNotExist {
+		return &iotextypes.BucketsCount{}, csr.Height(), nil
+	}
+	if err != nil {
+		return nil, 0, err
+	}
+	return &iotextypes.BucketsCount{
+		Total:  total,
+		Active: csr.ActiveBucketsCount(),
```

Comment:
for active count, similar to how to handle total amount in 
`func getTotalStakedAmount(ctx context.Context, csr CandidateStateReader) (*big.Int, error)`

Reply:
is that means we should have a func like `getActiveBucketsCount(ctx context.Context, csr CandidateStateReader) uint64` to return active count?